src/main/kotlin/tasks/Docker.kt

```diff
@@ -25,6 +25,9 @@ internal fun Project.pushedChartVersion(chartName: String) =
 internal fun Project.deployedChartFile(serviceName: String, namespace: String, chartName: String) =
     "$buildDir/deploy/${chartName}_${serviceName}_${namespace}_deployed_chart_values"
 
+// grype scan output
+internal fun Project.securityScanFile() = "$buildDir/scan/scan_report"
```

Comment:
The other output files are all located at `$buildDir/deploy/` so maybe `$buildDir/deploy/scan/scan_report`